slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -185,4 +215,11 @@ private long validatorId(final Bytes publicKey) {
     }
     return validatorId;
   }
+
+  private void lockForValidator(
+      final Handle handle, final LockType lockType, final long validatorId) {
+    final String lockTypePrefix = lockType == LockType.BLOCK ? "0" : "1";
```

Comment:
nit: you can potentially expose additional method in LockType enum, lockTypePrefix which can return appropriate prefix.

Reply:
maybe just used locktype.ordinal()?